packages/loki/spec/generic/dynamic_view.spec.ts

```diff
@@ -453,54 +453,11 @@ describe("dynamicviews", () => {
       let results = dv.data();
       expect(results.length).toBe(5);
       for (let idx = 0; idx < results.length - 1; idx++) {
-        expect(LokiOps.$lte(results[idx]["b"], results[idx + 1]["b"]));
+        expect(LokiOperatorPackageMap["js"].$lte(results[idx]["b"], results[idx + 1]["b"]));
       }
 
       // remove dynamic view
       coll.removeDynamicView("dvtest");
-
-      // add basic dv with filter on a and simplesort (with js fallback) on b
```

Comment:
Dynamic view tests are obsolete?

Reply:
yes, that one is... 
simplesort no longer has options for : 
- js fallback : now that we allow you to pass in the sortcomparer
- index intercept : no longer using this obsolete optimization based on binary indices (maybe if we rework data array to contain loki id's we can reintroduce.